test-suite/bugs/closed/bug_4287.v

```
@@ -104,15 +104,15 @@ Qed.
 End Hurkens.
 
 Polymorphic Record box (T : Type) := wrap {unwrap : T}.
-
+Set Printing Universes.
 (* Here we instantiate to Set *)
 
-Fail Definition down (x : Type) : Prop := box x.
+Definition down (x : Type) : Prop := box x.
```

This should fail, I think, at least with `Unset Universe Minimization ToProp ` or whatever.  Rigid universes should not be auto-minimized to `Prop`